### ISSUE TITLE
net.netfilter.nf_conntrack_max not applied on boot

### ISSUE THREAD

**Issue by ohrensessel** [ffnord/ffnord-puppet-gateway#119]
We are experiencing that net.netfilter.nf_conntrack_max is not applied on boot, so that we manually need to load /etc/sysctl.d/conntrack.conf

**Comment by ohrensessel** [comment]
do you see the same problem?

**Comment by sargon** [comment]
Nope. Usually the ``/etc/sysctl.d/*.conf`` files should be read by ``sysctl`` himself when called with ``--system``,
so when they aren't interpreted, either you have some value somewhere which is processed later or
for some reason the whole sysctl.d mechanism dosn't work on the problematic machine, maybe old version of sysctl?
